tuxemon/technique.py

```diff
@@ -99,6 +98,12 @@ def __init__(
         self.next_use = 0.0
         self.potency = 0.0
         self.power = 1.0
+        self.statspeed: Sequence[str] = []
```

Comment:
The typing is incorrect.  The JSON defines these values as a mapping/dict, not Sequence

Reply:
Not quite sure what you mean here, what would should I change it too?

Reply:
The thing is that in the JSON you have defined them to be a dictionary (a key-value mapping) and not a sequence.

In order to correctly type it you should create a typed dict class in `db.py`:
```python
class JSONStat(TypedDict):
    value: int  # I am not sure if this is supposed to be int or float, you decide
    dividing: bool
    overridetofull: bool
```
and use that to type the stat fields in `JSONTechniqueOptionalFields` and here.

Reply:
fixed it, but since that file isnt in this PR ill upload it in the next one

Reply:
The file IS in this PR, and you have already modified it. But you have repeated the typeddict definition in technique.py, try importing it from db.py.